.github/workflows/main.yml

```
@@ -1,23 +1,48 @@
+
+on:
+  pull_request:
+  push:
+    branches: [main]
+
 jobs:
-  jest-run:
+  unit-tests:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)